pkg/controller/deployment/generic_deployment_controller.go

```diff
@@ -64,7 +70,7 @@ func newGenericDeploymentController(
 			kubeInformersForTargetNamespace.Core().V1().Namespaces().Informer(),
 		},
 		controller,
-		openshiftClusterConfigClient,
+		&dummyClusterConfigClient{},
```

Comment:
IIUC, reading code from library-go seems like this config client is unused in terms of functionality within the controller, right?

[1] https://github.com/openshift/library-go/blob/master/pkg/operator/apiserver/controller/workload/workload.go

Reply:
Correspondingly, then we could replace `&dummyClusterConfigClient{}` with `dummyClusterConfigClient{}`, if that works (by removing the additional pointer reference) without causing any type issues.

Reply:
This gets completely removed in #85 [38c59a](https://github.com/openshift/cert-manager-operator/pull/85/commits/38c59a46e59f400ee1af924e58606b3590ad54bf).